caffe2/operators/c10_sigmoid_op.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "../core/tensor.h"
+#include "caffe2/utils/Array.h"
+
+namespace caffe2 {
+
+struct SigmoidOp final {
+    static constexpr const char* name = "sigmoid";
+    
+    using Signature = bool(Tensor<CPUContext> input, Tensor<CPUContext>* output);
```

Comment:
do you actually use the return value of bool anywhere? also, for c2 ops we kind of moved away from returning bools in general and prefer exceptions instead (there's still legacy code here and there left). So void should be just fine here.